eoc_journal/api_client.py

```diff
@@ -237,3 +248,20 @@ def get_cohort_average_progress(self):
         if data:
             return data.get('course_avg', None)
         return None
+
+    def get_user_proficiency(self):
+        """
+        Fetches and returns the user's and average course proficiency scores.
+        """
+        data = self._get_grades_leader_metrics()
+        if data is None:
+            return None
+
+        user_grade = data.get('user_grade')
+        course_avg = data.get('course_avg')
+        if user_grade is None or course_avg is None:
+            return None
+
+        user_grade = int(round(user_grade * 100.0))
+        course_avg = int(round(course_avg * 100.0))
```

Comment:
@bdero  Thanks for the suggestion.. I agree that's philosophically better, but I'm choosing to stick with the [calculation done by the original implementation](https://github.com/mckinseyacademy/mcka_apros/blob/751d04f999a417a4a74d7c380ccc65821601c0df/courses/views.py#L96).